src/chat.rs

```diff
@@ -3159,6 +3159,52 @@ pub async fn forward_msgs(context: &Context, msg_ids: &[MsgId], chat_id: ChatId)
     Ok(())
 }
 
+pub async fn resend_msgs(context: &Context, msg_ids: &[MsgId]) -> Result<()> {
+    let mut chat_id = None;
+    let mut msgs: Vec<Message> = Vec::new();
+    for msg_id in msg_ids {
+        let msg = Message::load_from_db(context, *msg_id).await?;
+        if let Some(chat_id) = chat_id {
+            ensure!(
+                chat_id == msg.chat_id,
+                "messages to resend needs to be in the same chat"
+            );
+        } else {
+            chat_id = Some(msg.chat_id);
+        }
+        ensure!(
+            msg.from_id == ContactId::SELF,
+            "can resend only own messages"
```

Comment:
this limits a bit the use case, when someone new is added to a group ideally any group member should be able to re-send the games etc. otherwise having to wait for the sender (that might have left the group etc)

but a good start anyway